sdk/servicebus/azure-servicebus/setup.py

```diff
@@ -63,7 +64,7 @@
         'azure',
     ]),
     install_requires=[
-        "uamqp>=1.5.1,<2.0.0",
+        "uamqp>=1.6.3,<2.0.0",
```

Comment:
you'll need to update the azure-sdk-for-python/shared_requirements.txt file so that:
- uamqp>=1.6.3
- The #override azure-servicebus statement has uamqp>=1.6.3

Reply:
I saw that and wondered if I needed to do that